src/BeamAdapter/component/BeamInterpolation.h

```diff
@@ -273,6 +272,12 @@ class BeamInterpolation : public virtual sofa::core::objectmodel::BaseObject
     void addCollisionOnBeam(unsigned int b) ;
     void clearCollisionOnBeam() ;
 
+    /////////////////////////// Deprecated Methods  ////////////////////////////////////////// 
+    unsigned int getNumBeamsNotUnderControl() { 

```

Comment:
You should add the [[deprecated]] keyword to warn the user at compile time too
(even if nobody actually reads the warnings 😅)

Reply:
Done